### ISSUE TITLE
About the partseg task

### ISSUE THREAD

**Issue by 964918993** [AnTao97/dgcnn.pytorch#22]
I found that every time I restarted the DGCNN-Partseg model, the train ACC was always the same from the beginning, although I didn't finish the training before retraining.  Do you know the reason for this strange phenomenon?
![runlog](https://user-images.githubusercontent.com/38918264/98675489-4c2c4100-2395-11eb-9db4-72d4073a5161.PNG)

Namespace(batch_size=16, class_choice=None, dataset='data/shapenetcore_partanno_segmentation_benchmark_v0', dropout=0.5, emb_dims=1024, epochs=300, eval=False, exp_name='partseg', k=20, lr=0.001, model='dgcnn', model_path='', momentum=0.9, no_cuda=False, num_points=1024, scheduler='cos', seed=1, test_batch_size=8, use_sgd=True)
Using GPU : 0 from 1 devices
Train 0, loss: 1.797249, train acc: 0.801439, train avg acc: 0.399633, train iou: 0.639207
Test 0, loss: 1.556620, test acc: 0.881248, test avg acc: 0.567119, test iou: 0.728525
Namespace(batch_size=16, class_choice=None, dataset='data/shapenetcore_partanno_segmentation_benchmark_v0', dropout=0.5, emb_dims=1024, epochs=300, eval=False, exp_name='partseg', k=20, lr=0.001, model='dgcnn', model_path='', momentum=0.9, no_cuda=False, num_points=1024, scheduler='cos', seed=1, test_batch_size=8, use_sgd=True)
Using GPU : 0 from 1 devices
Train 0, loss: 1.797249, train acc: 0.801439, train avg acc: 0.399633, train iou: 0.639207
Test 0, loss: 1.556620, test acc: 0.881248, test avg acc: 0.567119, test iou: 0.728525

**Comment by antao97** [comment]
Because the script use same seed (seed 1) to initialize model parameters. You can change it by adding `--seed new_seed_num` in training command.

**Comment by 964918993** [comment]
Sorry I didn't catch you. I tried to run with the command '--exp_name=partseg --seed=1" but it didn't solve the problem.
'--exp_name=partseg --seed=new_seed_num' didn't work either because the type isn't right.

Finally, I used '--exp_name= partseg --seed=2', and the problem was resolved and the training results changed.
But I still don't understand why. I didn't add the '--seed=new_sed_num(int)' command when I ran the classification task. Is training normal.

I'm wondering if I added the command you told me in the wrong place.
![捕获](https://user-images.githubusercontent.com/38918264/98787406-6c620b80-243a-11eb-98b6-93b2634c40e1.PNG)
